gmso/tests/test_mcf.py

```diff
@@ -0,0 +1,157 @@
+import pytest
+import mbuild
+import numpy as np
+import unyt as u
+
+import gmso
+from gmso.formats.mcf import write_mcf
+from gmso.tests.base_test import BaseTest
+from gmso.tests.utils import get_path
+from gmso.utils.io import get_fn
+from gmso.exceptions import EngineIncompatibilityError
+from gmso.external.convert_mbuild import from_mbuild
+
+
+class TestMCF(BaseTest):
+    def test_write_lj_mcf(self):
```

Comment:
Instead of creating a argon `Topology`, you can add the `typed_ar_system` fixture as an argument to this test to save yourself some lines of code.  Such as:

```
def test_write_lj_mcf(self, typed_ar_system):
    top = typed_ar_system
    write_mcf(top, 'ar.mcf')
```